src/components/products/ProductInfo.tsx

```
@@ -0,0 +1,146 @@
+                        `}
+                        disabled={quantity === 1}
+                    >
+                        −
+                    </button>
+
+                    <input
+                        type="text"
+                        value={quantity}
+                        onChange={(e) => {
+                            const val = parseInt(e.target.value);
+                            if (!isNaN(val)) setQuantity(Math.max(1, val));
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)